packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import classNames from 'classnames';
+import AccountPlatformIcon from '../../../components/account-platform-icon';
```

Comment:
@hirad-rewok for now i can't do that because account transfer component uses in appstore package and when i replace `import AccountPlatformIcon from '../../../components/account-platform-icon';` with import AccountPlatformIcon from 'Components/account-platform-icon'; and do npm run build:all command, build is failed because it cannot find this Component alias, the other files also imported using ../../../components.
<img width="441" alt="Screenshot 2023-01-24 at 17 01 55" src="https://user-images.githubusercontent.com/103181646/214314620-0ef576a6-4472-4bf8-ab88
<img width="1200" alt="Screenshot 2023-01-24 at 17 02 56" src="https://user-images.githubusercontent.com/103181646/214314882-a18802d2-0715-4fbb-96e3-4831d3e44530.png">
-9516fdb704bc.png">

Reply:
If I'm not mistaken we can not use aliases in cashier since we are going to export components in other packages

Reply:
@heorhi-deriv